### DISCUSSION TITLE
sending QR commands programatically

### DISCUSSION THREAD

**Discussion by rcleaves** [gopro/labs#502] [Q&A]
Is there a way to send a QR command (e.g. QR Command: oW1mVr1080!W!GMC) to a mini hero 11 black? Ideally, this would be done over wi-fi and use the HTTP API similar to the open GoPro SDK.

**Comment by dnewman-gpsw** [comment]
There is not, there should be, this is somewhat a feature request for Open GoPro (it is already requested).  You can ask over at https://github.com/gopro/OpenGoPro/discussions 

This reason for the separation is architectural, the camera has a low level RTOS (Real-time OS) for the fundamental camera functionality, and Linux kernel for networking functions.  Labs resides at the RTOS level.  Open GoPro runs of the Linux kernel, it would need an API to parse Labs messages into the RTOS.  I hope this happens in the not too distant future.

**Comment by dnewman-gpsw** [comment]
Follow up, wouldn't the commands you listed be directly supported by Open GoPro, without any Labs extensions?  Most requests for this is for Labs only functions like motion detection.
